code/modules/clothing/under/accessories/accessory_vr.dm

```diff
@@ -12,6 +12,37 @@
 	item_state = "choker_cst"
 	overlay_state = "choker_cst"
 	var/customized = 0
+	var/icon_previous_override
+	sprite_sheets = list(
+		SPECIES_TESHARI = 'icons/mob/species/seromi/ties_vr.dmi'
+		)
+
+//Forces different sprite sheet on equip
+/obj/item/clothing/accessory/choker/New()
+	..()
+	icon_previous_override = icon_override
+
+/obj/item/clothing/accessory/choker/equipped() //Solution for race-specific sprites for an accessory which is also a suit. Suit icons break if you don't use icon override which then also overrides race-specific sprites.
+	..()
+	setUniqueSpeciesSprite()
+
+/obj/item/clothing/accessory/choker/proc/setUniqueSpeciesSprite()
```

Comment:
Just tried to make this refitting business a bit more flexible so it ain't just hardcoded refit for Tesh only

Reply:
bet. This is just annoying to deal with and I'm way too tired and not even supposed to be doing dev work on the weekends for sanity.

Reply:
when you are you guys gonna backport my spicy ballistics code? :>

Reply:
@cadyn No clue, would be great to have but I'm almost solely focused on aesthetics and improving character-building

Reply:
Fair enough, pff, anyways thanks and you too <3